File: examples/TanksMiniGameProject/assets/unity/WXScripts.mgepackage/src/System/IO/Path.js
Bridge.assembly("minigame-adaptor-project", function ($asm, globals) {
    "use strict";

    Bridge.define("System.IO.Path", {
        statics: {
            fields: {
                DirectorySeparatorChar: 0
            },
            methods: {
                Combine: function (path1, path2) {
                    if (System.String.endsWith(path1, "/")) {
                        return (path1 || "") + (path2 || "");
                    }
                    return (path1 || "") + "/" + (path2 || "");
                },
                GetExtension: function (path) {
                    throw new System.Exception("not impl");
                },
                GetDirectoryName: function (path) {
                    throw new System.Exception("not impl");
                },
                GetFileNameWithoutExtension: function (path) {
                    throw new System.Exception("not impl");
                },
                GetFullPath: function (path) {
                    throw new System.Exception("not impl");
                },
                GetFullName: function (path) {
                    throw new System.Exception("not impl");
                },
                GetFileName: function (path) {
                    throw new System.Exception("not impl");
                }
            }
        }
    });
});

File: unity-plugin/script-export/sdk/System~/IO/Path.cs
namespace System.IO {
    public static class Path {
        public static string Combine(string path1, string path2) {
            if (path1.EndsWith("/")) {
                return path1 + path2;
            }
            return path1 + "/" + path2;
        }

        public static string GetExtension(string path) {
            throw new Exception("not impl");
        }

        public static string GetDirectoryName(string path) {
            throw new Exception("not impl");
        }

        public static string GetFileNameWithoutExtension(string path) {
            throw new Exception("not impl");
        }

        public static string GetFullPath(string path) {
            throw new Exception("not impl");
        }

        public static string GetFullName(string path) {
            throw new Exception("not impl");
        }

        public static string GetFileName(string path) {
            throw new Exception("not impl");
        }

        public static readonly char DirectorySeparatorChar;
    }
}